v1/runtime/runtime_test.go

```diff
@@ -1695,3 +1697,63 @@ func TestExtraDiscoveryOpts(t *testing.T) {
 		t.Error("expected extra discovery option to be called")
 	}
 }
+
+type h struct {
```

Comment:
[comment/nit]: I'm generally not a big fan of single-letter names (except in math equations!). I assume the actual names are not too significant, since they're strictly for test fixture bits?